test/runBTagAnalyzer_cfg.py

```diff
@@ -473,7 +473,7 @@
         postfix = postfix
     )
     getattr(process,'patJetPartonMatchCA8'+postfix).matched = cms.InputTag(genParticles)
```

Comment:
As far as I know, nobody is working on the CA8->AK8 switch. You cannot push because we switched to the _Fork & Pull_ development model (https://twiki.cern.ch/twiki/bin/view/CMS/BTagAnalyzer#Code_Development)